pkg/apis/kubecontrollermanager/v1alpha1/types.go

```diff
@@ -30,6 +30,11 @@ type KubeControllerManagerOperatorConfig struct {
 type KubeControllerManagerOperatorConfigSpec struct {
 	operatorsv1alpha1api.OperatorSpec `json:",inline"`
 
+	// forceRedeploymentReason can be used to force the redeployment of the kube-apiserver by providing a unique string.
+	// This provides a mechanism to kick a previously failed deployment and provide a reason why you think it will work
+	// this time instead of failing again on the same config.
+	ForceRedeploymentReason string `json:"forceRedeploymentReason"`
```

Comment:
possible values: "it will work now", "not it will surely work", "one more time"

Reply:
> possible values: "it will work now", "now it will surely work", "one more time"

Mine were
1. `!`
2. `!!`
3. `$%^&*!@^!`